uniflow/flow/config.py

```diff
@@ -217,9 +216,11 @@ class RaterClassificationOpenAIGPT4Config(RaterConfig):
     )
     guided_prompt_template: GuidedPrompt = field(
         default_factory=lambda: GuidedPrompt(
-            instruction="""Rate the answer based on the question and the context.
-        Follow the format of the examples below to include context, question, answer, and label in the response.
-        The response should not include examples in the prompt.""",
+            instruction="""
+            Evaluate the appropriateness of a given answer based on the question and the context.
+            There are few examples below, consist of context, question, answer, explanation and label.
+            Your response should only focus on the unlabeled sample, including two fields: label ("Yes" or "No") and explanation.
```

Comment:
The last part of your prompt `including two fields: label ("Yes" or "No") and explanation.` can be problematic especially, if user only change the label2score in the future without changing the prompt. Instead, we automatically added this postfix [here](https://github.com/Panzy-18/uniflow/blob/70a7fdf0072db17d2c623fc5b4b8b5e45114a9b1/uniflow/flow/config.py#L148) after __post_init__. You can consider modify this line in the __post_init__ if this prompt give you better performance.

Reply:
got it. I delete this.

Reply:
Another question I found: If users change label, they are also required to provide new labels' meaning and change the examples in guided prompt template.